src/mobot_client/models/__init__.py

```diff
@@ -1,65 +1,64 @@
-    OUT_OF_MOB = -2
-    CANCELLED = -1
-    READY_TO_RECEIVE_INITIAL = 0
-    WAITING_FOR_BONUS_TRANSACTION = 1
-    ALLOW_CONTACT_REQUESTED = 2
-    COMPLETED = 3
+from mobot_client.models.states import SessionState
 
 
 class Store(models.Model):
     name = models.TextField()
-    phone_number = models.TextField()
+    phone_number = models.TextField(db_index=True)
     description = models.TextField()
-    privacy_policy_url = models.TextField()
+    privacy_policy_url = models.URLField()
 
     def __str__(self):
         return f"{self.name} ({self.phone_number})"
 
 
 class Item(models.Model):
-    store = models.ForeignKey(Store, on_delete=models.CASCADE)
+    store = models.ForeignKey(Store, on_delete=models.CASCADE, related_name="items")
     name = models.TextField()
     price_in_pmob = models.PositiveIntegerField(default=None, blank=True, null=True)
     description = models.TextField(default=None, blank=True, null=True)
     short_description = models.TextField(default=None, blank=True, null=True)
-    image_link = models.TextField(default=None, blank=True, null=True)
+    image_link = models.URLField(default=None, blank=True, null=True)
+
+    @property
+    def price_in_mob(self) -> Decimal:
+        return mc.pmob2mob(self.price_in_pmob)
+
+    def available_skus(self) -> models.QuerySet:
+        return self.skus.filter(available__gt=0)
 
     def __str__(self):
         return f"{self.name}"
 
 
 class Sku(models.Model):
-    item = models.ForeignKey(Item, on_delete=models.CASCADE)
+    item = models.ForeignKey(Item, on_delete=models.CASCADE, related_name="skus")
     identifier = models.TextField()
     quantity = models.PositiveIntegerField(default=0)
     sort_order = models.PositiveIntegerField(default=0)
 
-    def __str__(self):
-        return f"{self.item.name} - {self.identifier}"
-
 
 class DropType(models.IntegerChoices):
     AIRDROP = 0, 'airdrop'
     ITEM = 1, 'item'
 
 
 class Drop(models.Model):
-    store = models.ForeignKey(Store, on_delete=models.CASCADE)
-    drop_type = models.IntegerField(choices=DropType.choices, default=DropType.AIRDROP)
+    store = models.ForeignKey(Store, on_delete=models.CASCADE, related_name='drops')
+    drop_type = models.IntegerField(choices=DropType.choices, default=DropType.AIRDROP, db_index=True)
     pre_drop_description = models.TextField()
-    advertisment_start_time = models.DateTimeField()
-    start_time = models.DateTimeField()
-    end_time = models.DateTimeField()
-    item = models.ForeignKey(Item, on_delete=models.CASCADE)
-    number_restriction = models.TextField()
-    timezone = models.TextField()
+    advertisment_start_time = models.DateTimeField(db_index=True)
+    start_time = models.DateTimeField(db_index=True)
+    end_time = models.DateTimeField(db_index=True)
+    item = models.ForeignKey(Item, on_delete=models.CASCADE, related_name='drops', db_index=True, null=True, blank=True)
+    number_restriction = models.TextField(default="+44")
```

Comment:
Nit: Conventions, the lines here use both `"` and `'`'. I suggest starting to use `black` and having CI enforce it. Takes away the need to have an opinion on which style to use and ensures consistency.